src/Plugin/WebformElement/CivicrmOptions.php

```diff
@@ -148,6 +148,11 @@ public function prepare(array &$element, WebformSubmissionInterface $webform_sub
     // A single static radio should be shown as a checkbox
     if (!$use_live_options && !$as_list && count($element['#options']) === 1) {
       $element['#type'] = 'checkbox';
+      // Reset the element label, the checkbox label, to be the first option's value.
+      $element['#title'] = reset($element['#options']);
+      // Remove the options array to prevent invalid validation.
+      // @see \Drupal\Core\Form\FormValidator::performRequiredValidation
+      unset($element['#options']);
```

Comment:
Keeping this around was causing the following logic check to pass

```
if (isset($elements['#options']) && isset($elements['#value'])) {
```

which caused the validation failure.